alembic/latiss/versions/60cee30b98da_add_column_scheduler_note.py

```diff
@@ -0,0 +1,152 @@
+"""Add column scheduler_note
+
+Revision ID: 60cee30b98da
+Revises: adbbeceff286
+Create Date: 2025-05-13 16:23:54.927825+00:00
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision: str = "60cee30b98da"
+down_revision: Union[str, None] = "adbbeceff286"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table_comment(
```

Comment:
Don't we need to do another `DROP VIEW`/`CREATE VIEW` sequence for each instrument?